### PR TITLE
Updated to Swift 4.2

### PR DESCRIPTION
**Pull request by freeubi**
I just updated the project to Swift 4.2. 
Cant run the tests for some reason, but the example app working well.

### CODE REVIEW

**Comment by AndriyGo** [comment]
I think that we'll wait with this PR. Xcode 10 is not yet public so merging this PR right now is impossible as it would break functionality.

**Comment by mduttondev** [comment]
@AndriyGo @Anarchoschnitzel Is there an ETA on getting this support for Swift 4.2 merged?

File: SCLAlertView.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -185,15 +195,17 @@
...
 				ORGANIZATIONNAME = "Alexey Poimtsev";
 				TargetAttributes = {
 					70264EF61B0F588700B32B18 = {
 						CreatedOnToolsVersion = 6.3.2;
+						DevelopmentTeam = 3GAGXCHTTV;
 						LastSwiftMigration = 0800;
 					};
 					70264F011B0F588800B32B18 = {
 						CreatedOnToolsVersion = 6.3.2;
+						DevelopmentTeam = 3GAGXCHTTV;
```
Comment (vikmeup) [inline SCLAlertView.xcodeproj/project.pbxproj]: can you remove DevelopmentTeam ?

**Comment by mduttondev** [comment]
I opened PR [#422](https://github.com/vikmeup/SCLAlertView-Swift/pull/422) in hopes of getting this merged sooner rather than later. 

Also this PR seems to be updating from Swift 3.0 which seems unneccessary since there is a released version that supports up to Swift 4.0

**Comment by freeubi** [comment]
as of https://github.com/vikmeup/SCLAlertView-Swift/pull/422, i'm closing this.

**Pull request closed without merging**